packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -326,30 +326,60 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
     AnnotationResult<S> result,
     Offset localPosition, {
     @required bool onlyFirst,
-  });
+  }) {
+    return false;
+  }
 
   /// Search this layer and its subtree for the first annotation of type `S`
   /// under the point described by `localPosition`.
   ///
   /// Returns null if no matching annotations are found.
   ///
   /// By default this method simply calls [findAnnotations] with `onlyFirst:
-  /// true` and returns the first result. It is encouraged to override
-  /// [findAnnotations] instead of this method.
+  /// true` and returns the annotation of the first result. It is encouraged to
+  /// override [findAnnotations] instead of this method.
```

Comment:
prefer "Prefer overriding [findAnnotations] rather than this method, because ..."